Casks/w/waveterm.rb

```diff
@@ -0,0 +1,25 @@
+cask "waveterm" do
+  version "0.5.1"
+  sha256 "fe965fe581a66bbd7c394dbfd7ebb1635cd0bf46329042ef86db123fbd7d5751"
+
+  url "https://dl.waveterm.dev/builds/waveterm-macos-universal-v#{version}.dmg"
+  name "Wave Terminal"
+  name "WaveTerm"
+  desc "Modern Terminal for Seamless Workflows"
+  homepage "https://www.waveterm.dev/"
+
+  livecheck do
+    url "https://github.com/wavetermdev/waveterm"
+    strategy :github_releases
+  end
```

Comment:
```suggestion
  livecheck do
    url "https://www.waveterm.dev/download"
    regex(/href=.*?waveterm.+[._-]v?(\d+(?:\.\d+)+)\.dmg/i)
  end
```

For the `livecheck` we should follow the source used for the `url`.

Reply:
ok, happy to change to the download page for livecheck.  just tested the regexp and updated it to this which now passes the audit:
`regex(/href=.*?dl\.waveterm\.dev.+-v(\d+(?:\.\d+)+)\.dmg/i)`

Reply:
Sorry, I missed part of the regex, I'll push a fix now.

Reply:
I don't have permission to push to the branch, so here's the suggestion.

```suggestion
  livecheck do
    url "https://www.waveterm.dev/download"
    regex(/href=.*?waveterm.+?[._-]v?(\d+(?:\.\d+)+)\.dmg/i)
  end
```

Reply:
No worries. Can you please just push this final suggestion as it aligns with our standard regex patterns more closely, and then we should be good to go.